src/main/resources/com/dubture/jenkins/digitalocean/SlaveTemplate/config.jelly

```diff
@@ -31,6 +32,10 @@
         <f:textbox/>
     </f:entry>
 
+    <f:entry title="Resolve user image by name rather than ID or slug" field="imageByName">
```

Comment:
I'm not against the checkbox, but I think its a bit messy for user. Can jelly files create a row so you have a dropdown?
Legacy (ID or Slug) | ByID | BySlug | ByName